lib/OpenQA/Schema/Result/Users.pm

```diff
@@ -51,6 +51,14 @@ __PACKAGE__->add_columns(
         false_id      => ['0', '-1'],
         default_value => '0',
     },
+    last_login_version => {
+        data_type   => 'text',
+        is_nullable => 1,
+    },
+    feature_informed => {
```

Comment:
@coolo i want to do this in this area:
`  last_login_version => {
        data_type   => 'text',
        is_nullable => 1,
    },`
the boolean part is  just for checking if the user is informed about the newest feature.
So lets say there is feature01, feature02, ...feature05 in the last_login_version (maybe i should change the description). The jquery code will check for the version number, and if the user saw the latest feature, the boolean part will set to true. Is this the right direction?